test/e2e/nodemaintenance_test.go

```diff
@@ -74,6 +76,28 @@ func showDeploymentStatus(t *testing.T, f *framework.Framework) {
 	t.Fatalf("operator logs: %s", str)
 }
 
+const (
+	scriptShowCRD = `./kubevirtci/cluster-up/kubectl.sh get -o yaml crd nodemaintenances.kubevirt.io`
```

Comment:
I don't like that we're using shell here, but the programatic approaches investigated so far have too many dependancies.

Reply:
Removed the changes from the golang e2e tests. Now all the test are run as part of functest.sh